### DISCUSSION TITLE
About searching performance with opensearch in jaeger

### DISCUSSION THREAD

**Discussion by hueiyuan** [jaegertracing/jaeger#4259] [Q&A]
## Description
Hi everyone. I would like to ask and check a problem. Because we currently use jaeger to search data and backend storage is opensearch in AWS. But we found when we search performance from opensearch is too slow so that always encounter timeout issue with jaeger. And data always write to opensearch at the same time.

We have adjust the number of shard and replicas in opensearch. But still have the same problem. Therefore, does anyone had the same problem in past? If you have any idea, please share your thought or solution. I will be appreciated it, thank you.

**Comment by jkowall** [comment]
Tuning and scaling OpenSearch is a tricky topic. I would suggest engaging with AWS support or the OpenSearch forums you can find on opensearch.org. It's not really something we handle in the Jaeger project specifically.
